edc-tests/edc-controlplane/agreement-retirement-tests/src/test/java/org/eclipse/tractusx/edc/tests/agreement/retirement/RetireAgreementTest.java

```diff
@@ -131,6 +131,12 @@ void retireAgreement_shouldCloseTransferProcesses() {
 
         }
 
+        @Test
+        @DisplayName("Verify non existing Contract Agreement is not stored in retired table.")
+        void retireAgreement_shouldFailIfTryingToRetireNonExistingId() {
```

Comment:
the correct format is eventually:
```retireAgreement_shouldFail_whenAgreementDoesNotExist```

Reply:
thank you, you're right, is more explicit. changed in [here](https://github.com/eclipse-tractusx/tractusx-edc/pull/1631/commits/b54f8c3a7cd2e664633b06e778c1cb5d6bf32c09).